### ISSUE TITLE
require-prop-comment

### ISSUE THREAD

**Issue by czb3279338858** [vuejs/eslint-plugin-vue#1992]
**Please describe what the rule should do:**

prop should have a comment

**What category should the rule belong to?**

[ ] Enforces code style (layout)
[ ] Warns about a potential error (problem)
[X] Suggests an alternate way of doing something (suggestion)
[ ] Other (please specify:)

**Provide 2-3 code examples that this rule should warn about:**

```vue
      <template>
        <div>1</div>
      </template>
      <script>
      import { defineComponent } from '@vue/composition-api'

      export default defineComponent({
        props: {
          a: Number,
          // b comment
          b: Number,
          // c
          // comment
          c: Number
        }
      })
      </script>
```

```vue
      <template>
        <div>1</div>
      </template>
      <script setup>
      const props = defineProps({
        a: Number
      })
      </script>
```

**Additional context**

**Comment by czb3279338858** [comment]
Duplication due to network latency
